### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by DGuhr** [dasniko/testcontainers-keycloak#51] [General]
Just wanted to let a thank you here for providing the tools devs want for the new keycloak ;) so again: thanks! 

Much appreciated, i even think about using it some time in the future in the main testsuite, where we actually also use testcontainers for some tests now.  cc @andreaTP ;)

**Comment by dasniko** [comment]
Thanks, @DGuhr, awesome! I really appreciate that! 🙏

Would love to see Testcontainers-Keycloak even used by Keycloak itself. 😃
